python/src/viewer.py

```diff
@@ -524,6 +527,8 @@ def convRGBA(c):
             legend_kwargs.setdefault('prop', {'size': 10})
 
             self._ax[0].legend(**legend_kwargs)
+        if square_axes:
+            self._ax[0].axis('square')
 
```

Comment:
Seems this is not supported by older matplotlib (see the autobuilder log), maybe use something like this:
```
try:
    self._ax[0].axis('square')
except ValueError:
    plt.gca().set_aspect('equal', adjustable='box')
```